site/code/index.html.js

```diff
@@ -53,6 +53,13 @@ In addition to the GraphQL [reference implementations in JavaScript](#javascript
 using System;
 using GraphQL;
 using GraphQL.Types;
+using GraphQL.SystemTextJson;
+
+// You can use GraphQL.NewtonsoftJson with .NET Core 3+, just be aware it lacks async writing 
+// capabilities so writing to an ASP.NET Core 3.0 HttpResponse.Body will require you to set 
+// AllowSynchronousIO to true which isn't recommended.
```

Comment:
```
AllowSynchronousIO ...
```

This comment pertains to ASP.NET Core, which isn't this example of a Console application.  I don't think we need to add this comment here.